war/postcss.config.js

```diff
@@ -1,6 +1,7 @@
 module.exports = {
   parser: "postcss-less",
   plugins: [
-    require('autoprefixer')
+    require('autoprefixer'),
+    require('postcss-custom-properties')
```

Comment:
so reading up on what this does. Doesn't custom-properties plugin remove all the variables and replace them with the actual values?

Reply:
No it doesn’t remove them it adds the fallbacks above the variables for browsers that don’t support them (IE)

The theme wouldn’t work at all if it did it in the way you described